src/stratigraphy/benchmark/score.py

```diff
@@ -27,7 +49,9 @@ def f1(precision: float, recall: float) -> float:
         return 0
 
 
-def evaluate_matching(predictions_path: Path, ground_truth_path: Path) -> tuple[dict, pd.DataFrame]:
+def evaluate_matching(
+    predictions_path: Path, ground_truth_path: Path, directory: Path, out_directory: Path
```

Comment:
these new params are missing at the bottom of this file

Reply:
Addressed. Thank you.